src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -904,29 +906,37 @@ import ageDisplay, { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
     * Creates color steps for the heatmap.
     * @name InCHlib#color_steps
     */
-    self.color_steps = [
+    self.color_steps = (
+      under = self._get_color_for_value()(0),
+      middle = self._get_color_for_value()(0.5),
+      over = self._get_color_for_value()(1),
+    ) => [
```

Comment:
generate the colour gradients dynamically, based on passed values.
This ensures 3 step gradients are centred, rather than just amalgamated.